ambassador/ambassador/ir/irambassador.py

```diff
@@ -36,7 +36,8 @@ class IRAmbassador (IRResource):
         'x_forwarded_proto_redirect',
         'load_balancer',
         'xff_num_trusted_hops',
-        'server_name'
+        'server_name',
+        'circuit_breakers'
```

Comment:
Once upon a time, these were sorted. They should be sorted again. :)

Reply:
Fixed :+1: